.github/workflows/e2e_test.yml

```diff
@@ -30,4 +30,5 @@ jobs:
           with:
             go-version: 1.19
 
-        - run: make e2e-test-ibc
\ No newline at end of file
+        - run: make e2e-test-ibc
+        - run: make e2e-test-ibc-timeout
```

Comment:
Can we run them in parallel or with the current architecture this is not possible right now?

Reply:
We could actually go for
```
- run: make e2e-test-all
```

Reply:
I understand your current concern regarding the issue of parallel execution. I'll create another PR to incorporate them into a feasible matrix architecture for parallel CI execution. @pgoos